.github/workflows/build.yml

```diff
@@ -0,0 +1,128 @@
+#
+# Copyright 2021 The Sigstore Authors.
```

Comment:
nit: could we keep the .yaml extension here and on all the other github files to make history tracking/review easier?

Reply:
@dlorenc Yeah agree. I will review other changes too to avoid breaking history.

Reply:
@dlorenc Done, let me know if you need anything else.